client/webserver/site/src/js/wallets.js

```diff
@@ -13,7 +13,6 @@ export default class WalletsPage extends BasePage {
     super()
     app = application
     this.body = body
-    this.notifiers = {}
```

Comment:
It's defined down below. https://github.com/decred/dcrdex/pull/601/commits/d3bc9cb9eb71b35de24181f1b75547c97d4dd646#diff-f8404ab8f5958f186ea0c27ebdc9f6efL137